core/store/src/main/java/uk/gov/gchq/gaffer/store/StoreProperties.java

```diff
@@ -72,6 +73,19 @@
 
     public static final String ADMIN_AUTH = "gaffer.store.admin.auth";
 
+    /**
+     * This is used....
```

Comment:
I presume these comments need completing.

Reply:
It doesn't look like that to me. Requires clarification.